app/src/main/java/com/nervousfish/nervousfish/data_objects/RSAKey.java

```diff
@@ -0,0 +1,62 @@
+package com.nervousfish.nervousfish.data_objects;
+
+/**
+ * RSA variant of {@link IKey}.
+ */
+public final class RSAKey implements IKey {
+
+    private final static String TYPE = "RSA";
```

Comment:
Indeed! I already used it [here](https://github.com/ericcornelissen/NervousFish/blob/app/database/contacts/app/src/main/java/com/nervousfish/nervousfish/modules/database/GsonKeyAdapter.java#L36) and [here](https://github.com/ericcornelissen/NervousFish/blob/app/database/contacts/app/src/main/java/com/nervousfish/nervousfish/modules/database/GsonKeyAdapter.java#L79)